src/transformers/utils/generic.py

```diff
@@ -336,6 +336,23 @@ def __exit__(self, *args, **kwargs):
         self.stack.__exit__(*args, **kwargs)
 
 
+def can_return_loss(model_class):
+    """
+    Check if a given model can return loss.
+
+    Args:
+        model_class (`type`): The class of the model.
+    """
+    model_name = model_class.__name__
+    if model_name.startswith("TF"):
+        signature = inspect.signature(model_class.call)
+    elif model_name.startswith("Flax"):
+        signature = inspect.signature(model_class.__call__)
+    else:
+        signature = inspect.signature(model_class.forward)
+    return [p for p in signature.parameters if p == "return_loss"]
```

Comment:
I'd prefer if this returns a `bool` then rely on Python magic conversion to bools.
Also I think we should check if the default is `True` as the Trainer won't change the default.

Reply:
I changed it to 
```python
any(p == "return_loss" for p in signature.parameters)
```
but do you mean we should have sth. like (conceptually)

```pythont 
any(p == "return_loss" and default_value(p) is True for p in signature.parameters)
```
?

Reply:
Yes to the second!